### DISCUSSION TITLE
Please support Uno 4

### DISCUSSION THREAD

**Discussion by sigmarsson** [PrismLibrary/Prism#2708] [Uno Platform]
1. When do you publish [the Uno 4 relevant changes](https://github.com/PrismLibrary/Prism/blob/master/Packages.props#L14-L15) in Prism ? 

2. If you act so, will you upgrade ```DryIoc.dll``` to 5 as well ?

Thank you.

**Comment by dansiegel** [comment]
We don't publish release timeframes. When the next public release of Prism goes out it will have the latest version of DryIoc. 

If you need access sooner I would suggest either becoming a GitHub Sponsor to access the CI Feed. Or you can get an Enterprise Support contract at [avantipoint.com](https://avantipoint.com)
